app/Http/Controllers/Api/V1/LtiTool/LtiToolSettingsController.php

```diff
@@ -71,10 +84,14 @@ public function show(Organization $suborganization, $id)
 
     /**
      * Create Lti tool Setting
+     * 
      * Creates the new lti tool setting in database
+     * 
      * @param StoreLtiToolSetting $request
      * @param Organization $suborganization
```

Comment:
Does not seem to be valid format for suborganization in in LtiToolSettingsController.php.

Reply:
Done!